src/models/propulsion/FGTurbine.cpp

```diff
@@ -488,9 +485,9 @@ bool FGTurbine::Load(FGFDMExec* exec, Element *el)
   if (tsfcElement) {
     string value = tsfcElement->GetDataLine();
     if (is_number(value))
-      TSFC = std::make_unique<FGRealValue>(atof(value.c_str()));
+      TSFC = std::make_unique<FGTSFC>(this, atof(value.c_str()));
     else
-      TSFC = std::make_unique<FGFunction>(FDMExec, tsfcElement, to_string((int)EngineNumber));
+      TSFC = std::make_unique<FGTSFC>(this, FDMExec, tsfcElement, EngineNumber);
```

Comment:
Just use the plain `FGFunction` class in that case.
```suggestion
      TSFC = std::make_unique<FGFunction>(FDMExec, tsfcElement, to_string((int)EngineNumber));
```